test/CMakeLists.txt

```diff
@@ -133,6 +133,7 @@ set(MULTI_TEST_SRC
     backend_arg_reduce.in.cpp
     backend_test.in.cpp
     backend_unary_elementwise.in.cpp
+    backend_nnp_datatype_test.in.cpp
```

Comment:
bfloat16 is not an nnp datatype. nnp does use bfloat16 but we should treat it as just another datatype. Upcoming CPUs support bfloat16. The file name should be something different or these tests should be included in other files.

Reply:
We also have some float16 hardware, so we need to be careful with naming.